components/profile-controller/pkg/apis/istio/v1alpha1/doc.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```

Comment:
quick question: for the header, should we use kubeflow version or kubernetes version?  I've seen them here and there...

Reply:
For Istio types, since they are generated from Istio CRD, `kubernetes ` or `istio` might be more accurate than `kubeflow`.
@jlewi wdyt? any concerns?